pika/adapters/twisted_connection.py

```diff
@@ -105,35 +114,79 @@ def channel_closed(self, _channel, reason):
         self.__calls = set()
         self.__consumers = {}
```

Comment:
I'm not the original author of that but if I understand correctly the point was to avoid conflicting with one of the wrapped method (since `__getattr__` is subclassed here). However none of the wrapped method start with an underscore and I don't think we will want to wrap one that does in the future, so a single underscore prefix should be sufficient. I also don't expect the `TwistedChannel` class to be subclassed. I'll change it.